editor/configs/map.ts

```diff
@@ -1,18 +1,20 @@
-import { LngLatBoundsLike } from "react-map-gl";
+import { LngLatLike } from "react-map-gl";
```

Comment:
I didnt catch this earlier (or maybe it got edited later i dunno), but i dont see where this is being used